pom.xml

```diff
@@ -31,10 +31,12 @@
         <version.org.jboss.logging>3.6.0.Final</version.org.jboss.logging>
         <version.org.apache.commons.commons-lang3>3.14.0</version.org.apache.commons.commons-lang3>
         <version.org.apache.httpcomponents.httpclient>4.5.14</version.org.apache.httpcomponents.httpclient>
+        <version.org.bouncycastle>1.76</version.org.bouncycastle>
```

Comment:
Are the versions of the added dependencies aligned with WildFly Core (or WildFly)?

Reply:
Looks like I'm a bit behind with the WildFly Core, updated now